code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -410,7 +412,7 @@
 					remove_object(O)
 					break
 		//Plants
-		else if(istype(I, /obj/item/grown))
+		if(istype(I, /obj/item/grown))
```

Comment:
this else made sense

Reply:
No, not really. I was a bit overzealous in removing it, I don't think it fixes any bugs. I can put it back, it doesn't change anything.